modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -185,6 +207,10 @@ class Instrument_List extends \NDB_Menu_Filter
 
                 $Ins = "instruments";
 
+                if ($isSurvey == '1' && !$isSent) {
```

Comment:
Should the value for `$isSurvey` be expected as a string?

Reply:
You're right, it's a `tinyint(1)`

Reply:
It's `tinyint(1)` in the DB but `gettype($isSurvey)` returns string. I think this check is fine.